apps/styleguide/src/display/LinksList.vue

```diff
@@ -32,12 +32,19 @@ export default {
 <docs>
 Display a list of links
 ```
-const links = [
-    { label: 'Molgenis', href: 'https://molgenis.org', },
-    { label: 'Vue',  href: 'https://vuejs.org/' } 
-]
+<script>
+  export default {
+    data() {
+      return {
+        links: [{label: 'Molgenis', href: 'https://molgenis.org'}]
+      }
+    }
+  }
```

Comment:
this makes me a bit sad , is all the js vue needed to define a example ?, what issue is being fixed by this change ?

Reply:
no, that is indeed not needed in this case. Can be example in one line.